### DISCUSSION TITLE
Any mkdocs-material syntax ignored in md files

### DISCUSSION THREAD

**Discussion by noeldum** [squidfunk/mkdocs-material#6516] [Q&A] [answered]
I am probably facing a simple problem but still need help for this.

Any mkdocs-material specific syntax that I include in my md files are ignored. i.e. They appear as just text in the rendered page.

Here is a dummy project setup to test this. I am on ubuntu.
mkdocs-material verion is 9.4.3
mkdocs version is 1.5.3
python version is 3.10.12

I just created a project
`mkdocs new .`

Edited the config:
```
site_name: My Docs
theme:
  name: material
```

Added something mkdocs-material specific in index.md. A button here.
```
# Welcome to MkDocs
[Subscribe to our newsletter](#){ .md-button }
```

Running this below seems ok.
`mkdocs serve`

Then my page looks like this
![image](https://github.com/squidfunk/mkdocs-material/assets/1875345/daa993b4-a8e4-4048-8332-35faf961795d)

Any help appreciated. Am I missing something? It seems straight forward enough.

**Comment by kamilkrzyskow** [comment, marked as answer]
Hello,
for the `{ .md-button }` syntax to work you need to enable the `attr_list` markdown extension, and the `{}` syntax is not exclusive for mkdocs-material, only the `md-button` class is directly related to it.

```yaml
site_name: My Docs
theme:
  name: material
markdown_extensions:
  - attr_list
```

https://squidfunk.github.io/mkdocs-material/reference/buttons/#configuration
https://python-markdown.github.io/extensions/attr_list/
